frontend/test/specs/institutionServiceSpec.js

```diff
@@ -87,9 +87,9 @@
         it('Test searchInstitution', function() {
             var documents = [{name: institutions[0].name, id: institutions[0].key}];
             spyOn($http, 'get').and.callThrough();
-            httpBackend.expect('GET', "/api/search/institution?value=" + '"' + institutions[0].name + '"' + "&state=active" ).respond(documents);
+            httpBackend.expect('GET', "/api/search/institution?value=" + institutions[0].name + "&state=active&type=institution" ).respond(documents);
```

Comment:
padrão de query params do angular 

https://stackoverflow.com/questions/13760070/angularjs-passing-data-to-http-get-request